apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/b_Builders/ProxyResponseBuilder.py

```
@@ -28,6 +29,9 @@
             print_logs(json.dumps({"statusCode": status_code, "body": json.loads(body)}, default=str))
     except:
         print_logs('Log failed', default=str)
+    headers = {
```

Ruff (0.11.9)

Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)